test/test.js

```diff
@@ -66,6 +66,14 @@ describe('PostHTML-Parser test', function() {
         expect(parser('<!doctype html>')).to.eql(['<!doctype html>']);
     });
 
+    it('should be parse directive', function() {
```

Comment:
IMHO, need some test case when `<?php ...>` nested into any html tag (for example, `<div>`).

Reply:
Very thanks, i find bug, and fixed him. )

Reply:
`<?php wp_nav_menu(array('theme_location' => 'primary', 'menu_class' => 'header-nav-bar')); ?>`
is not working correctly

Reply:
@IgorMotorny Hi, I need more information. Create a separate Issue and describe the problem case. Thanks.